src/main/java/com/github/seikichi/pronto/infer/example/App.java

```diff
@@ -10,4 +10,22 @@ public String getGreeting() {
   public static void main(String[] args) {
     System.out.println(new App().getGreeting());
   }
+
+  public int test1() {
+    String s = null;
+    return s.length();
```

Comment:
object `s` last assigned on line 15 could be null and is dereferenced at line 16.